test/box/gh-6405-discarding-arguments-in-box-func-call.result

```diff
@@ -0,0 +1,190 @@
+-- test-run result file version 2
+env = require('test_run')
+ | ---
+ | ...
+test_run = env.new()
+ | ---
+ | ...
+
+test_run:cmd("create server test with script = \"box/lua/cfg_test1.lua\"")
+ | ---
+ | - true
+ | ...
+test_run:cmd("start server test")
+ | ---
+ | - true
+ | ...
+test_run:cmd("switch test")
+ | ---
+ | - true
+ | ...
+
+sum_args = function(...)                      \
+    local type_table = {}                     \
+    for i = 1, select('#', ...) do            \
+    type_table[i] = type(select(i, ...))      \
+    end                                       \
+    return type_table                         \
+end                                           \
+
+function TablesEquals(table1, table2)         \
```

Comment:
I would name it `TablesAreEqual`. Or `tables_are_equal`.

Reply:
fixed